### DISCUSSION TITLE
Failing remote build with no logs

### DISCUSSION THREAD

**Discussion by Z-100** [vercel/community#3245] [Help] [answered]
### Summary

Hello there

I have a React app, which uses npm. It's hosted on GitHub and uses Vercel as a deployment pipeline. It uses the default commands (`npm i`, `npm run build`.

Until yesterday everything worked fine: Merge feature -> develop branch, vercel does vercel-stuff, the changes are live. Merge develop -> main, vercel-stuff, changes are in prod.

After I've added postcss & a prestart script (`"prestart": "postcss ./src/css/app.css -o ./src/App.css"`) however, the develop build failed without any reason or what so ever. The only thing displayed in Vercel's logs were the following:

```text
// Building tab
Running build in Cleveland, USA (East) – cle1
Cloning [repository] (Branch: develop, Commit: [commit])
Cloning completed: 453.038ms
Restored build cache
Running "vercel build"
Vercel CLI 31.0.2
Running "install" command: `npm i`...
```

Now to the actual weird thing though: I've installed the CLI tool, built the project via `vercel build` & deployed it via `vercel` and suddenly... everything worked without any error. The deployment worked and is currently live.

### Steps to Reproduce

1. Deploy `develop`-branch on vercel website
    - Either by pushing something to it
    - Or hitting "redeploy"
2. See it failing
3. Try same thing using Vercel CLI
    - enter `vercel build`
    - then `vercel`
4. See it working

If anyone could help, I'd greatly appreciate that. Thanks!

**Comment by amyegan** [comment]
If you share the deployment id, then I can look into our logs for more details

**Reply by Z-100** [reply, marked as answer]
Hi there. Thanks for your reply!
It's not really needed anymore, as I figured out the problem: I had a "prestart" script, but vercel uses the build and not start script. I just had to add a "prebuild" script.
So basically vercel didn't fail me. My React knowledge did :D
